tests/lib/index.test.ts

```diff
@@ -32,21 +32,122 @@ describe("index node tests", () => {
 
 
         const board = new Board();
-        const kit = board.addKit(WeaviateKit);
-
-        kit
-            .index()
-            .wire("dataFile<-", board.input())
-            .wire("weaviateHost<-", board.input())
-            .wire("PALM_KEY<-palmApiKey", board.input())
-            .wire("className<-", board.input())
-            .wire("->totalIndexedDocuments", board.output());
-
-        const results = await board.runOnce(inputs);
-        expect(results.totalIndexedDocuments).toBe(docCount);
+        const kit: WeaviateKit = board.addKit(WeaviateKit);
+
+        const index = kit.index();
+
+        const input = board.input({
+            schema: {
+                type: "object",
+                properties: {
+                    dataFile: {
+                        type: "string",
+                    },
+                    weaviateHost: {
+                        type: "string",
+                    },
+                    className: {
+                        type: "string",
+                    },
+                    palmApiKey: {
+                        type: "string",
+                    },
+                },
+            },
+        });
+        input.wire("dataFile", index);
+        input.wire("weaviateHost", index);
+        input.wire("className", index);
+        input.wire("palmApiKey->PALM_KEY", index);
+
+        index.wire("*", board.output());
```

Comment:
I don't understand this way of building a board. Could you please link to the docs in breadboard that discuss this approach?